yazi-cli (ya.exe) doesnt compile on windows

### What system are you running Yazi on?

Windows

### What terminal are you running Yazi in?

n/a

### `yazi --debug` output

```Shell
n/a
```

### Describe the bug

```
error[E0432]: unresolved import `windows_sys::Win32::Storage::FileSystem::CreateFileW`
   --> yazi-term\src\tty\handle.rs:134:139
    |
134 | ... Storage::FileSystem::{CreateFileW, FILE_SHARE_READ, FILE_SHARE_WRITE, OPEN_EXISTING}, System::Console::Get...
    |                           ^^^^^^^^^^^ no `CreateFileW` in `Win32::Storage::FileSystem`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `yazi-term` (lib) due to 1 previous error
```

### Minimal reproducer

`cargo install --path yazi-cli`

### Anything else?

_No response_

### Checklist

- [x] I tried the [latest nightly build](https://yazi-rs.github.io/docs/installation#binaries), and the issue is still reproducible
- [x] I updated the debug information (`yazi --debug`) input box to the nightly that I tried
- [x] I can reproduce it after disabling all custom configs/plugins (`mv ~/.config/yazi ~/.config/yazi-backup`)

--- a/yazi-term/Cargo.toml
+++ b/yazi-term/Cargo.toml
@@ -27,4 +27,4 @@ libc = { workspace = true }
 crossterm = { workspace = true, features = [ "use-dev-tty", "libc" ] }
 
 [target.'cfg(windows)'.dependencies]
-windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading", "Win32_Security"] }
+windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading", "Win32_Security" ] }

--- a/yazi-term/Cargo.toml
+++ b/yazi-term/Cargo.toml
@@ -27,4 +27,4 @@ libc = { workspace = true }
 crossterm = { workspace = true, features = [ "use-dev-tty", "libc" ] }
 
 [target.'cfg(windows)'.dependencies]
-windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading" ] }
+windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading","Win32_Foundation", "Win32_Security"] }

--- a/yazi-term/Cargo.toml
+++ b/yazi-term/Cargo.toml
@@ -27,4 +27,4 @@ libc = { workspace = true }
 crossterm = { workspace = true, features = [ "use-dev-tty", "libc" ] }
 
 [target.'cfg(windows)'.dependencies]
-windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading","Win32_Foundation", "Win32_Security"] }
+windows-sys = { version = "0.59.0", features = [ "Win32_Globalization", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_Console", "Win32_System_Threading", "Win32_Security"] }

